Tools/Scripts/libraries/webkitscmpy/webkitscmpy/remote/bitbucket.py

```diff
@@ -322,40 +339,171 @@ def comments(self, pull_request):
                         content=comment.get('text'),
                     )
 
+        @classmethod
+        def _position_converters(cls, json_diff):
+            abosolute_to_relative = dict(
+                source=defaultdict(lambda: defaultdict(None)),
```

Comment:
is `defaultdict(lambda: defaultdict(None))` any different from a `defaultdict(dict)`?